app/main.py

```diff
@@ -1,8 +1,47 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: str) -> None:
```

Comment:
When you make a line brake after `__ini__(`, corresponding `)` should start from the new line. Place `) -> None:` on a new line.